examples/mugen/retrieval/train.py

```diff
@@ -47,7 +47,8 @@ def train():
 
     trainer = Trainer(
         accelerator=args.accelerator,
-        devices=1,
+        devices=args.devices,
+        strategy="ddp_find_unused_parameters_false",
```

Comment:
How do we know there's no unused parameter?

Reply:
when i didn't have this argument, the script printed a warning saying no unused params were found and having `find_unused_parameters=True` makes performance worse